example/lib/polyline_annotations_example.dart

```diff
@@ -30,15 +31,20 @@ class PolylineAnnotationExampleState extends State<PolylineAnnotationExample> {
       polylineAnnotationManager = value;
       createOneAnnotation();
       final positions = <List<Position>>[];
-      for (int i = 0; i < 99; i++) {
+      for (int i = 0; i < 500; i++) {
```

Comment:
500 might be too many 😅 

<img width="1206" height="2622" alt="Image" src="https://github.com/user-attachments/assets/2270ec7a-74ba-4dd7-9214-17ab91b875e8" />

Reply:
Sorry, checked only with circle annotations example and 500 seemed reasonable there, put the numbers back